Samples/outlook-set-signature/README.md

```diff
@@ -3,14 +3,16 @@ title: "Set your signature using Outlook event-based activation"
 page_type: sample
 urlFragment: outlook-add-in-set-signature
 products:
-- office-outlook
-- office
+  - office-outlook
+  - office
+  - office-365
+  - office-teams
```

Comment:
confirming that this sample includes JSON manifest...?

Reply:
Yes, this sample includes a JSON manifest.